recipe/meta.yaml

```
@@ -221,7 +226,7 @@ outputs:
       license: BSD-3-Clause
 
   - name: vs_{{ cross_target_platform }}
-    version: {{ vsyear }}.{{ update_version }}
+    version: {{ vsyear }}.{{ update_version|int }}
```

The `|int` doesn't really make sense, since we want a string here.
You could try
```suggestion
    version: '{{ vsyear }}.{{ update_version }}'
```
to see whether you could circumvent some unintended YAML number parsing from happening.

---

Thanks for the tip! I was kind of stabbing in the dark here - I had suspected things go wrong on insert, I didn't even consider that it gets _reread_ as YAML again.